LICENSE

```diff
@@ -1,201 +1,16 @@
-                                 Apache License
-                           Version 2.0, January 2004
-                        http://www.apache.org/licenses/
+Apache Software License 2.0
 
-   TERMS AND CONDITIONS FOR USE, REPRODUCTION, AND DISTRIBUTION
+Copyright (c) 2019, Sherin Thomas
```

Comment:
Change to `Copyright (c) 2019, Tensorwerk Inc, Sherin Thomas` unless @lantiga would like to specify this differently?

Reply:
Sure thing. Sorry for being sloppy here. This is done by the cookie cutter